qa/cephfs/conf/mds.yaml

```diff
@@ -4,6 +4,10 @@ overrides:
       mds:
         debug mds: 20
         debug ms: 1
+        mds bal fragment size max: 10000
+        mds bal merge size: 5
+        mds bal split bits: 3
+        mds bal split size: 100
```

Comment:
if the option of `mds_bal_frag` has been dropped, and the related options are standard and folded into `qa/cephfs/conf/mds.yaml`, can we just drop `qa/cephfs/overrides/frag_enable.yaml` ?

Reply:
Oh, I thought I had. I'll fix that.